Detectors/EMCAL/base/src/Geometry.cxx

```diff
@@ -10,20 +10,38 @@
 // or submit itself to any jurisdiction.
 #include "EMCALBase/Geometry.h"
 
+#include <RtypesCore.h>
+#include <TMath.h>
+#include <TVector3.h>
+#include <TMathBase.h>
+#include <TVector2.h>
+#include <TParticle.h>
```

Comment:
why do we need to include all of these classes? TParticle for example should not be used in the Geometry, right?

Reply:
In my clean up PR that I am working on, I at least try to remove things like:
```c++
phi = TVector2::Phi_0_2pi(phi); // move phi to (0,2pi) boundaries
```
and instead use the proper O2 tools for this. Same with TMath.

Reply:
Okay